main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -209,7 +209,7 @@ object Main {
           Bootstrap.bootstrap(cwd)(System.out) match {
             case Result.Ok(bootstrap) =>
               val result = bootstrap.test(o)
-              System.exit(getCode(result))
+              System.exit(toCode(result))
```

Comment:
But if we are always under an OK branch, we might as well, just call exit(0)

Reply:
I think the Bootstrap is Ok but then the method called on the Bootstrap object can also fail (within the Ok branch)